cypress/e2e/pageObjects/OrangeHRMAddEmployee/assertions.ts

```diff
@@ -36,9 +36,17 @@ class assertions{
         cy.get("span").contains("Should have at least 7 characters").should('be.visible');
     }
 
-    checkTheEmployeeRecordNotAppear(id:string){
+    checkTheEmployeeRecord(employeeRow: [id : string , firstName : string , lastName : string], isExist: boolean){
         cy.wait(7000);
```

Comment:
Why you need static wait here ? ,if there is any intercept you should use it instead